src/index.js

```diff
@@ -42,4 +35,31 @@ const reload = () => {
   }
 };
 
-window.onload = reload();
\ No newline at end of file
+window.onload = reload();
+
+const task = document.getElementById('add-item');
```

Comment:
- [ ] OPTIONAL
Your input element is accepting empty fields. Kindly ensure to validate this input to check that the input is not empty before data is stored.

**Your project looks like this, when empty details is entered:**
![image](https://user-images.githubusercontent.com/12745474/130175427-97f5cf63-3956-4232-b8db-a7c5b29e653f.png)

Reply:
"Add to your list..." `input` now requires at least a character. Required change completed.